### PR TITLE
added div() method

### PR DESCRIPTION
**Pull request by asrdigi**
Please review my feature branch, div_f and approve

### CODE REVIEW

File: src/com/sapient/asde/service/Calculator.java [hunk 1]
```diff
@@ -12,4 +12,8 @@ public double sub(double number1,double number2){
     public double mul(double number1,double number2){
         return number1*number2;
     }
+
+    public double div(double number1,double number2){
+        return number1/number2;
```
Comment (advith-2022) [inline src/com/sapient/asde/service/Calculator.java]: apply exception-handling if number2 is 0

**Review by advith-2022 — COMMENTED** [review]
Update the div() method

File: src/com/sapient/asde/service/Calculator.java [hunk 2]
```diff
@@ -14,6 +14,14 @@ public double mul(double number1,double number2){
     }
 
     public double div(double number1,double number2){
-        return number1/number2;
+        try {
+            if(number2==0.0){
```
Comment (advith-2022) [inline src/com/sapient/asde/service/Calculator.java:18]: this code is fine

**Review by advith-2022 — APPROVED** [review]
Approved to merge div_f to master branch

**Comment by asrdigi** [comment]
div_f merged to master branch

**Pull request merged**